docs/remote/linux.md

```diff
@@ -21,8 +21,19 @@ The extensions are known to work with recent stable/LTS version of:
 
 There is also experimental support in **[VS Code Insiders](https://code.visualstudio.com/insiders/)** for:
 
-* **Ubuntu ARMv8l (AArch64) 64-bit SSH hosts** (18.04+) in Remote - SSH
-* **Windows 10 / Server 2016/2019 SSH hosts** (1803+) using the [official OpenSSH Server](https://docs.microsoft.com/windows-server/administration/openssh/openssh_install_firstuse).
+SSH Hosts (Remote - SSH):
+
+* **Ubuntu ARMv8l (AArch64) 64-bit** (18.04+)
+* **Windows 10 / Server 2016/2019** (1803+) using the [official OpenSSH Server](https://docs.microsoft.com/windows-server/administration/openssh/openssh_install_firstuse).
+
+Containers (Remote - Containers)
```

Comment:
Both ARM architectures will be available in Stable for Remote-Containers. The log terminal with show the same "is in preview" message as it shows for Alpine Linux since not all extensions will cope well with these platforms.

@aeschli @roblourens Are Remote-WSL and Remote-SSH enabling both ARM architectures in Stable too?

Reply:
Yes